src/renderer/components/Dashboard.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <el-container class="dashboard-container">
+    <aside class="aside">
+      <div class="sidemenu">
+        <div class="sidemenu__top">
+          <router-link class="sidemenu__item" to="/dashboard/summary-page">
+            <i class="el-icon-menu sidemenu__icon"></i>
```

Comment:
I put [an ElementUI's icon](https://element.eleme.io/#/en-US/component/icon#icon) as a dummy for now.